### DISCUSSION TITLE
All seem to be OK, Windhawk not uninstalled, but the mods not charged... ?

### DISCUSSION THREAD

**Discussion by sypqys** [ramensoftware/windhawk-mods#578] [Q&A] [answered]
Hello

I almost uninstalled Windhawk by mistake but I cancelled, when I restart it, it doesn't seem to work even though everything is active...
Why is this? What can I do?
![explorer_YqhnOqs4zP](https://github.com/ramensoftware/windhawk-mods/assets/27967268/d2feb647-bb83-42e0-abea-c917f43b0826)

Thanks a lot!

**Comment by m417z** [comment]
The screenshot seems fine, so it's difficult to say what's wrong from the information you provided. Did you customize anything in the advanced settings? Perhaps you excluded some of the processes? If not, uninstall and re-install Windhawk (settings and mods will be kept) and see if it helps.

**Reply by sypqys** [reply]
I have restarted the computer, and it seems to be OK. All rework fine ! Thanks !

**Comment by sypqys** [comment]
I have check : in order to hide icon, but when I attempt to restard Windhawk, it doesn't open the mods...

Why ?

I have to restart the computer each time.

![VSCodium_vthwh20FQb](https://github.com/ramensoftware/windhawk-mods/assets/27967268/c6e04b4e-5ce8-4552-b543-ac2eea72a9f8)

**Reply by m417z** [reply]
Do you mean that after you click on the blue "Restart Windhawk" button, the mods no longer work?
First of all, you don't need to click on it if you don't change any of the settings on that page.
In any case, it should continue to work, not sure why it doesn't. You can open Task Manager, find Windhawk in services, and check if it's running or not. Maybe it can't start for some reason.

**Reply by sypqys** [reply]
I have kill task of Windhawk first of all, and restart the soft, nothing mod charged. And the same closing VSCodium, but anything happens when I restart the software.

**Reply by m417z** [reply, marked as answer]
I really don't know what to tell you without being able to reproduce it. I'm not even sure I understand exactly what you're doing. Can you post a video recording with your exact actions?
